qemu/tests/balloon_check.py

```diff
@@ -193,9 +193,8 @@ def get_memory_boundary(self, balloon_type=''):
             self.wait_for_balloon_complete(balloon_timeout)
             self.ori_gmem = self.get_memory_status()
         else:
-            vm_total = self.get_memory_status()
             vm_mem_free = self.get_free_mem()
-            used_size = vm_total - vm_mem_free + 16
+            used_size = self.ori_mem - vm_mem_free + 16
```

Comment:
According to kvm developer, besides used memory and crashkernel memory size, we should also keep 300M for safety.  Actually 300M should work, but sometimes it also depends on the setup.  For now we could keep 300M if testing passed after 10 rounds.  Could we enlarge the buffer here?

Reply:
Sure.
Modified.
But guest mem is very small, such as 512M, used_size maybe larger than self.ori_mem. So we suggest limit linux guest minimum mem to 1G.
@huangyum Do you agree?